generators/python_functions/most_frequent_words/code_snippet.md

```diff
@@ -0,0 +1,11 @@
+```python
+from collections import Counter
+
+YOUR_ATTRIBUTE: str = "text"  # only text fields
+YOUR_N_WORDS: int = 5
+
+def most_frequent_words(record):
+    
+    words = [token.text for token in record[YOUR_ATTRIBUTE] if not token.is_stop and not token.is_punct]
+    return Counter(words).most_common(YOUR_N_WORDS)
```

Comment:
This returns a list of tuples, which we cannot work within refinery at the moment. Doing it like this worked for me: `return str(Counter(words).most_common(YOUR_N_WORDS)).strip("[]")`

Reply:
Yes, I'll test that thoroughly and fix it :)